### ISSUE TITLE
Dual Version

### ISSUE THREAD

**Issue by 9H1LO** [bg3mdo/budget_23cm_band_RF_power_amplifier#3]
Very nice project, how is the progress with the dual ldmos version ?

**Comment by bg3mdo** [comment]
I will update it soon

**Comment by 9H1LO** [comment]
many thanks looking forward to this

**Comment by wb0gaz** [comment]
Also interested dual LDMOS version!

**Comment by wb0gaz** [comment]
Also interested in dual-device QRO version, open source (gerbers) or other way. Thank you to BG3MDO + BG0AUB for nice innovation.

**Comment by 9H1LO** [comment]
any updates on publishing or it is a dead project ?

**Comment by 9H1LO** [comment]
> Also interested dual LDMOS version!

yes still waiting @wb0gaz  but OM @bg3mdo  not reply hope he is all okay and FB !

**Comment by wb0gaz** [comment]
Thank you 9H1LO, still no update. Now MRF13750H part is end of life, so QRO option for 23CM PA is reduced, 4 X MRFE6S9160 would be good option for 23CM EME!
